webapp/components/test/test-search.html

```diff
@@ -396,6 +396,19 @@
       });
     });
 
+    test('simple triaged search', () => {
+      assertQueryParse('triaged:chrome', { exists: [{ triaged: 'chrome' }] });
+    });
+
+    test('test status | triaged', () => {
```

Comment:
Isn't this an 'and', not an 'or'? Perhaps I misunderstand what you meant by 'test status | triaged' - I suggest giving tests more descriptive names.